etc/profile-a-l/firefox-common.profile

```diff
@@ -34,7 +34,7 @@ include whitelist-var-common.inc
 
 apparmor
 caps.drop all
-# machine-id breaks pulse audio; add it to your firefox-common.local if sound is not required.
+# machine-id breaks pulse audio; add it to your firefox-common.local if pulse audio is not being used.
```

Comment:
- PulseAudio ?
- Do we expect readers of this comment to know what PA is?
- In which reasonable scenario is sound not used for browsers?

Reply:
Note that there are similar comments in different profiles:

```console
$ git show --pretty='%h %ai %s' -s
fd5659e43 2022-08-28 22:00:36 +0200 tuir.profile creation (#5330)
$ git grep '# machine-id ' -- etc
etc/profile-a-l/enpass.profile:# machine-id and nosound break audio notification functionality.
etc/profile-a-l/ephemeral.profile:# machine-id breaks pulse audio; it should work fine in setups where sound is not required.
etc/profile-a-l/firefox-common.profile:# machine-id breaks pulse audio; add it to your firefox-common.local if sound is not required.
etc/profile-m-z/thunderbird.profile:# machine-id breaks audio in browsers; enable or put it in your thunderbird.local when sound is not required
etc/profile-m-z/yelp.profile:# machine-id breaks sound - add the next line to your yelp.local if you don't need sound support.
```

Suggestion: Remove this change from this PR and fix/standardize all
relevant comments later in a new PR.

Reply:
@rusty-snake [on Aug 28](https://github.com/netblue30/firejail/pull/5343#discussion_r956771450):

> * In which reasonable scenario is sound not used for browsers?

I would guess that the point of the `s/sound/pulse audio/` change is that
multiple sound servers exist, but the comment only makes claims about
pulseaudio specifically.

Would be nice to know if `machine-id` affects alsa/sndio/pipewire as well.

Reply:
Only PA is broken by `machine-id`, not even pipewire-pulse.

PA is still the most common sound server (default in all major distros except Fedora). So for a user w/o much technical knowledge is PA=sound. Therefore it may be good to hint users that PA is for audio output/input. If you use alsa/jack/... you likely opted-in to it and know that you use it but if you just use the default install of Mint, do you know the name of some background deamon that just does its job?

Reply:
As it's not enabled by default there should be no problem. I don't think someone will enable it without knowing "what PA is ?".